### ISSUE TITLE
mljsadmin capture throwing error

### ISSUE THREAD

**Issue by dmcassel** [adamfowleruk/mlnodetools#23]
I installed using `sudo npm install -g https://github.com/adamfowleruk/mlnodetools`. I then tried to capture, but got errors:

```
mljsadmin --capture
```

> An uncaught exception occurred: TypeError: this._driver is not a function
> FATAL ERROR: TypeError: this._driver is not a function
> - Check mljsadmin.log for details

Where is mljsadmin.log? 

```
mljsadmin --capture=dbconfig
```

> An uncaught exception occurred: TypeError: this._driver is not a function
> FATAL ERROR: TypeError: this._driver is not a function
> - Check mljsadmin.log for details

Same error with `mljsadmin --capture -h`.

**Comment by adamfowleruk** [comment]
Looks like a low level bug in the backend connection class wrapper. Can you do me a favour and sneak a look at the node_modules/mljs/package.json file underneath where mlnodetools is installed? I suspect it's a dependency issue. Details on platform would be good too.

Thanks.

**Comment by dmcassel** [comment]
package.json:

```
{
  "name": "mlnodetools",
  "description": "Command line tools for working with MarkLogic Server",
  "bugs": {
    "url": "https://github.com/adamfowleruk/mlnodetools/issues",
    "email": "adam.fowler@marklogic.com"
  },
  "author": {
    "name": "Adam Fowler",
    "email": "adamfowleruk@gmail.com",
    "url": "http://www.adamfowler.org/"
  },
  "license": "Apache-2.0",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/adamfowleruk/mlnodetools.git"
  },
  "version": "8.0.12",
  "bin": {
    "mljsadmin": "./src/util/mljsadmin.js",
    "mljsserve": "./src/util/run-mljs-webserver.js"
  },
  "main": "./src/util/mlnodetools.js",
  "dependencies": {
    "chalk": "^1.1.3",
    "istextorbinary": "^1.0.0",
    "minimist": "^1.1.0",
    "q": "^1.1.2",
    "winston": "^0.8.3"
  },
  "gitHead": "0e7186614e812ec43dd2463e87502dcaacfcf585",
  "readme": "# mlnodetools\n\nThis project provides command line tools to simplify administration of MarkLogic Server Enterprise NoSQL installations,\nand applications built against them.\n\nThis project provides the below tools:-\n- mljsadmin - installs, updates, deploys, captures, and tears down entire MarkLogic Server applications and databases\n- mljsserve - runs an Express based web app that also proxies /v1/ REST API URLs to a MarkLogic Server database app server (For use in DEV and DEMO situations ONLY - exposes the entire /v1/ set of REST endpoints to the web app user - so not for production use!)\n\n## Recent changes\n2016-Apr-08 - V. 8.0.11 - More reliable folder loading with threading. Support for file permissions. Reliability improvements. Colour coding in terminal window. (No Windows support)\n\n## Installing mlnodetools\n\nWARNING: You MUST install mlnodetools GLOBALLY:-\n\n1. npm install -g mlnodetools\n2. If you receive permission warnings, follow the advice on the following page, and then try the above command again: https://docs.npmjs.com/getting-started/fixing-npm-permissions\n3. Test by typing just 'mljsadmin' with no parameters from the command line (this MUST be issued in the root folder of your app)\n\n## Other tools\n\nIn order to create a new MLJS Workplace based web application project, use the companion Yeoman mljsworkplace generator.\n\nFor details and how to use this command, please visit: https://github.com/adamfowleruk/generator-mljsworkplace\n\n## Administering a MarkLogic Server application\n\nOnce generated, you simply:-\n- Open a terminal\n- cd to your new project folder\n- type 'mljsadmin' for a list of commands you can run at the root of that application\n\n## Examples\n\nBelow are a few examples\n\n### Installing a new application\n\nIf you've just downloaded an MLJS Workplace based application, execute these commands to fully install it:-\n\n- mljsadmin install\n- mljsadmin update\n\nIf you see any WARN or ERROR lines, follow the advice that mljsadmin gives to resolve them, before continuing.\n\n### Capturing an existing application's settings\n\nIf you've created or modified a new application, reconfigured a database, or modified your workplace pages or application\nontology then you need to capture the configuration of your system in order to share the full application with others.\n\nThis is achieved by:-\n\n- mljsadmin capture\n\nAgain, correct any WARN or ERROR messages, if required\n\n### Removing an application\n\nTo remove all databases, triggers, app servers and so on from your MarkLogic installation, you need to remove your\napplication:-\n\n- mljsadmin remove\n\n### Other commands\n\nTo see a full list of commands, type:-\n\n- mljsadmin\n\n## Logging bugs\n\nPlease visit http://github.com/adamfowleruk/mlnodetools/issues in order to log bugs and feature requests\n",
  "readmeFilename": "README.md",
  "homepage": "https://github.com/adamfowleruk/mlnodetools#readme",
  "_id": "mlnodetools@8.0.12",
  "_shasum": "ada8a9776fef494d0910422e486696820140215d",
  "_from": "git+https://github.com/adamfowleruk/mlnodetools.git",
  "_resolved": "git+https://github.com/adamfowleruk/mlnodetools.git#0e7186614e812ec43dd2463e87502dcaacfcf585"
}
```

Platform:
- Mac OS X 10.10.5
- Node v4.3.0

**Comment by adamfowleruk** [comment]
Thanks Dave. This should be fixed in 8.0.13. Note you will also have to 'npm install -g mljs' before executing mljsadmin again also.
